ext/intl/calendar/calendar_methods.cpp

```diff
@@ -156,16 +156,6 @@ U_CFUNC PHP_FUNCTION(intlcal_get_keyword_values_for_locale)
 	}
 
 	//does not work; see ICU bug 9194
-#if 0
```

Comment:
Better convert this to comment. Removal makes the preceding comment look like the code after if does not work, which is wrong.